buildSrc/src/main/java/com/osacky/flank/gradle/results/UniqueName.kt

```diff
@@ -0,0 +1,57 @@
+package com.osacky.flank.gradle.results
+
+import com.osacky.flank.gradle.fladleDir
+import org.gradle.api.Project
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+import java.util.Random
+
+fun createDefaultResultsDir(project: Project, name: String): String {
+  val configPath = project.layout.fladleDir.map {
+    if (name == "") {
+      it
+    } else {
+      it.dir(name.toLowerCase(Locale.ROOT))
+    }
+  }.get().asFile.absolutePath
+  return configPath + "/results/${uniqueObjectName()}"
+}
+
+private fun uniqueObjectName(): String {
```

Comment:
can we just use `UUID.randomUUID().toString()`? or is the date important for some reason?

Reply:
Sure we can, I just copy-pasted implementation from flank, just for testing

Reply:
oh i see. in that case do we want to use a similar name? or can we delegate the creation of this to flank somehow?

Reply:
> or can we delegate the creation of this to flank somehow

That is rather not possible

> in that case do we want to use a similar name?

That was my intention, but it can be whatever you want. I'd like to leave date + time, that's for sure. Rest is up to you

Reply:
if there direction we are going towards is to make this task cacheable, we should not be generating unique output directories. ( I know there is some other ongoing work to make this task cacheable)

I think the reason to have the unique output directory is so that the history can be kept? Is that the use case?

Reply:
Ah, I see your concern. I think this implementation is incorrect then. Need to change it